### ISSUE TITLE
any chance for huawei device in the future ?

### ISSUE THREAD

**Issue by ggmo** [kevinbowen777/config-compliance#7]
any chance for huawei device in the future ?

**Comment by kevinbowen777** [comment]
Certainly. If someone provided a pull request and gave me access to a handful of Huawei configs, I woulod definitely take a look at it see about merging it. I, personally, haven't used Huawaei devices and do not have any configs to reference or test with. Thanks for your interest.
